advertools/spider.py

```diff
@@ -652,6 +653,7 @@ def __init__(self, url_list, follow_links=False,
                  xpath_selectors=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.start_urls = json.loads(json.dumps(url_list.split(',')))
+        self.meta = eval(json.loads(json.dumps(meta)))
```

Comment:
Yes. 

This is where the `meta` dictionary should be saved and accessed when needed whenever requests are made.